common/src/main/java/com/microsoft/identity/common/internal/controllers/NativeAuthMsalController.kt

```diff
@@ -0,0 +1,63 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.common.internal.controllers
+
+import com.microsoft.identity.common.java.commands.parameters.nativeauth.AcquireTokenNoFixedScopesCommandParameters
+import com.microsoft.identity.common.java.commands.parameters.nativeauth.BaseNativeAuthCommandParameters
+import com.microsoft.identity.common.java.logging.LogSession
+import com.microsoft.identity.common.java.providers.nativeauth.NativeAuthOAuth2Strategy
+import com.microsoft.identity.common.java.providers.oauth2.OAuth2StrategyParameters
+import com.microsoft.identity.common.java.result.AcquireTokenResult
+import lombok.EqualsAndHashCode
+
+/**
+ * The implementation of MSAL Controller for Native Authentication.
+ */
+@EqualsAndHashCode(callSuper = true, onlyExplicitlyIncluded = true)
+class NativeAuthMsalController : BaseNativeAuthController() {
+
+    companion object {
+        private val TAG = NativeAuthMsalController::class.java.simpleName
+    }
+    
+    fun acquireTokenSilent(
+        parameters: AcquireTokenNoFixedScopesCommandParameters
+    ): AcquireTokenResult {
+        LogSession.logMethodCall(TAG, "${TAG}.acquireTokenSilent")
+
+        val acquireTokenSilentResult = AcquireTokenResult()
+        
+        return acquireTokenSilentResult
+    }
+    
+    private fun composeOAuth2Strategy(parameters: BaseNativeAuthCommandParameters): NativeAuthOAuth2Strategy {
```

Comment:
Is this method needed? I don't see it being used anywhere

Reply:
Any reason for using the word "compose" here as opposed to "create"? "create" is being used in all the existing code.

Reply:
I have changed it.